rustica/src/verification.rs

```diff
@@ -15,6 +15,17 @@ pub fn verify_piv_certificate_chain(
     client: &[u8],
     intermediate: &[u8],
 ) -> Result<Key, RusticaServerError> {
+    // Restrict the max size of certificates
+    // For Yubikey 5 Nano, actual intermediate cert size is approx 800 bytes
+    // and actual client cert size is approx 700 bytes
+    const CERT_MAX_SIZE: usize = 1024 * 2; // 2 KiB
```

Comment:
These should go at the top of the file or in the lib.rs

Reply:
@obelisk Do you think the size check itself should be done in sshcerts too? That'd be cleaner but it might break some edge cases where people have unusually large attestation data.